docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -38,7 +38,7 @@ The `CREATE TABLE` statement is used to create a new table in a keyspace. It def
 ```
 create_table ::= CREATE TABLE [ IF NOT EXISTS ] table_name
                      '(' table_element [ ',' table_element ...] ')'
-                     [ table_option [ AND table_option ] ];
+                     [WITH table_options];
```

Comment:
```suggestion
                     [WITH table_properties];
```
So that it matches with `table_properties` description section below on this page.